exercises/concept/windowing-system/.docs/hints.md

```diff
@@ -42,16 +42,16 @@
 - Remember to use [class syntax][mdn-class] for this task.
   That means you add the method definition directly in the class body.
 - To find the correct values for the moving, first think about the desired `x` position you want to reach, taking the lower bound (`0`) into account.
-- Then calculate the maximum `x` value that is allowed considering the screen size and current width of the window.
+- Then, calculate the maximum allowed `x` value. Consider the screen size and the current width of the window.
```

Comment:
Not sure if the new sentence needs to be on a separate line

Reply:
It should be